### PR TITLE
#167296792: Create spinner component

### PR DESCRIPTION
**Pull request by Tyak99**
### What does this PR do?

* Create spinner component

### Description of Task to be completed?

* add spinner component
* write test for spinner component
* remove `@babel/polyfill` import from index file

### Any background context you want to provide?

The spinner component can be used to let the user know their request is loading.
image

### What are the relevant Github issues?

N/A

### Questions:

N/A

### CODE REVIEW

**Review by Ochowo — APPROVED** [review]

**Review by Ochowo — COMMENTED** [review]
Looks good

**Comment by chekwas88** [comment]
LGTM

File: src/components/Spinner/spinner.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+  @keyframes lds-ring {
+    0% {
+      transform: rotate(0deg);
+    }
+    100% {
+      transform: rotate(360deg);
+    }
+  }
+
```
Comment (fortunee) [inline src/components/Spinner/spinner.scss]: New line here

**Review by fortunee — CHANGES_REQUESTED** [review]
Just a wee fix

**Review by fortunee — APPROVED** [review]
LGTM

**Pull request merged**